web/frontend/src/components/keyboard-input.vue

```diff
@@ -74,9 +82,16 @@ const removeKeyboardListeners = () => {
   window.removeEventListener('keyup', onUseKeyboardNav);
 };
 
-const toggleKeyboard = () => {
+const stopBase = (name: string) => {
+  const req = new baseApi.StopRequest();
+  req.setName(name);
```

Comment:
I'd recommend just using `props.name` here, so that we aren't passing it around throughout this component.